self_hosting_machinery/webgui/selfhost_statistics.py

```diff
@@ -85,11 +77,8 @@ async def streamer():
         except Exception as e:
             raise HTTPException(status_code=500, detail=str(e))
 
-    async def _dash_prime_get(self):
-        if not self._stats_service.is_ready:
-            return self._stats_service_not_available_response
-
-        data_tables = await compose_data_frames(self._stats_service)
+    async def _dash_prime_get(self, request: Request):
+        data_tables = await self._stats_service.compose_data_frames(request)
```

Comment:
I dislike passing request in stats_service

Reply:
fixed